### PR TITLE
fix: 1599 task updated related resources modal

### PR DESCRIPTION
**Pull request by tangopapatime**
# Description
Updated related resource modal

image

### CODE REVIEW

File: packages/client/hmi-client/src/services/data.ts [hunk 1]
```diff
@@ -446,15 +446,15 @@ const getAssets = async (params: GetAssetsParams) => {
  * the "/documents?similar_to=<id>" endpoint as an alternative
  */
 const getRelatedDocuments = async (docid: string, dataset: string | null) => {
```
Comment (shawnyama) [inline packages/client/hmi-client/src/services/data.ts:448]: Rename `docid` to `projectId`?

File: packages/client/hmi-client/src/components/dataset/tera-dataset.vue [hunk 2]
```diff
@@ -256,14 +256,18 @@
 								<span>Suggested value</span>
 								<div>
 									<div class="suggested-value-source">
-										<i class="pi pi-file" />{{ dataset.metadata?.documents[0].title }}
+										<i class="pi pi-file" />{{
+											dataset.metadata?.documents ? dataset.metadata?.documents[0].title : ''
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/dataset/tera-dataset.vue:260]: Nitpick: you can make it shorter - `dataset.metadata?.documents?.[0].title ?? ''`

File: packages/client/hmi-client/src/components/dataset/tera-dataset.vue [hunk 3]
```diff
@@ -277,7 +281,7 @@
 						</div>
 					</div>
 				</AccordionTab>
-				<AccordionTab v-if="pd.length > 0">
+				<AccordionTab v-if="pd && pd.length > 0">
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/dataset/tera-dataset.vue:284]: Nitpick: `!isEmpty(pd)` could check for whether it is exists and if length > 0

File: packages/client/hmi-client/src/components/widgets/tera-related-publications.vue [hunk 4]
```diff
@@ -31,6 +32,17 @@
 					<Column field="name" sortable header="Name"></Column>
 					<Column field="authors" sortable header="Authors"></Column>
 				</DataTable>
+				<div v-else class="no-artifacts">
+					<div class="no-artifacts">
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/widgets/tera-related-publications.vue:36]: Repeated div

File: packages/client/hmi-client/src/services/data.ts (hunk 1 above)
Comment (tangopapatime) [reply packages/client/hmi-client/src/services/data.ts:448]: docid matches the api call for now.

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/components/widgets/tera-related-publications.vue [hunk 5]
```diff
@@ -147,4 +146,24 @@ const sendForEnrichments = async (/* _selectedResources */) => {
...
+	flex-direction: column;
+	align-items: center;
+}
+
+.no-artifacts-img {
+	width: 30%;
+	padding: 10px;
+}
+
+.no-artifacts-text {
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/widgets/tera-related-publications.vue:161]: You don't need specialize class when the parent as one.
`.no-artifacts img` would suffice

Comment (tangopapatime) [reply packages/client/hmi-client/src/components/widgets/tera-related-publications.vue:161]: @YohannParis I know.  I personally like to use a dedicated class vs doing something like that.  honestly don't know what's actually the preferred way.

**Pull request merged**